scripts/helm/helm_handle_chart.sh

```diff
@@ -19,14 +19,20 @@ on_exit () {
 }
 trap "{ on_exit; }" EXIT
 
-# Check for Before Deploy Scripts
-bash $SCRIPTS_DIR/deploy/before-deploy.sh "$HELM_CHART_DIRECTORY"
 
 # Load bitops.config.yaml
 export BITOPS_CONFIG_COMMAND="$(ENV_FILE="$BITOPS_SCHEMA_ENV_FILE" DEBUG="" bash $SCRIPTS_DIR/bitops-config/convert-schema.sh $BITOPS_CONFIG_SCHEMA $HELM_BITOPS_CONFIG)"
 echo "BITOPS_CONFIG_COMMAND: $BITOPS_CONFIG_COMMAND"
 source "$BITOPS_SCHEMA_ENV_FILE"
 
+if [ -n "$SKIP_DEPLOY" ]; then
```

Comment:
For ENV changes, I've been prefixing the deployment tool name. In this case it should be `HELM_SKIP_DEPLOY`. 

`SKIP_DEPLOY` is too generic and would be confusing for BitOps users to provide this value manually or could create weird bugs for BitOps developers if the variable was not scoped properly 

Can you update https://github.com/bitovi/bitops/blob/master/scripts/helm/bitops.schema.yaml#L41 as well?